kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -427,6 +433,21 @@ public static ConfigDef newConfigDef() {
           "S3 Proxy Password"
       );
 
+      configDef.define(
+          HEADERS_USE_EXPECT_CONTINUE_CONFIG,
+          Type.BOOLEAN,
+          HEADERS_USE_EXPECT_CONTINUE_DEFAULT,
+          Importance.LOW,
+          "Enable/disable use of the HTTP/1.1 handshake using EXPECT: 100-CONTINUE during "
+              + "multi-part upload. If true, the client will wait for a 100 (CONTINUE) response "
+              + "before sending the request body. Else, the client uploads the entire request "
+              + "body without checking if the server is willing to accept the request.",
+          group,
+          ++orderInGroup,
+          Width.SHORT,
+          "S3 Proxy Use Expect Continue"
```

Comment:
nit: I believe proxy is not relevant now. 
`S3 HTTP Send Uses Expect Continue`?